gluten-core/src/main/scala/io/glutenproject/execution/WriteFilesExecTransformer.scala

```diff
@@ -47,6 +47,7 @@ import scala.collection.convert.ImplicitConversions.`collection AsScalaIterable`
  */
 case class WriteFilesExecTransformer(
     child: SparkPlan,
+    override val outputOrdering: Seq[SortOrder],
```

Comment:
can we move it into class instead of constructor ? it should also be `child.outputOrdering`

Reply:
RemoveNativeWriteFilesSortAndProject will remove sort child before FileFormatWriter#write, so the FileFormatWriter#outputOrderingMatched will not be set.

Reply:
How about override `executeAndCheckOrdering` ? Velox table write does not require ordering, the `hasLogicalSort` and `hasEmpty2Null` should always be false.